<launch>


<!-- Launching USB CAM launch files and Dynamixel controllers -->

  <include file="$(find eyecu)/launch/usb_cam/start_usb_cam.launch"/>


<!-- Starting face tracker node -->
   <rosparam file="$(find eyecu)/config/track.yaml" command="load"/>

   <node name="eyecu" pkg="eyecu" type="eyecu_node_dlib" output="screen"/>

</launch>
